app/index.html

```diff
@@ -42,7 +42,19 @@
               <li><a ui-sref="classify">Rating game</a></li>
               <li><a ui-sref="about">About</a></li>
             </ul>
-            <ul class="nav navbar-nav hide-collapsed">
+
+            <ul class="nav navbar-nav navbar-right player-controls">
```

Comment:
`float: right` works better if its before the centered elements. Therefore i swapped these two `<ul>` blocks.